ds3_java_cli/src/main/java/com/spectralogic/ds3cli/Main.java

```diff
@@ -32,7 +33,13 @@
     public static void main(final String[] args) {
         try {
             final Arguments arguments = new Arguments(args);
+
             final Ds3Client client = createClient(arguments);
+            if (!Utils.isCliSupported(client)) {
+                System.out.println(String.format("ERROR: Minimum Black Pearl supported is %s", Utils.MINIMUM_VERSION_SUPPORTED));
+                System.exit(2);
```

Comment:
I used the same status code like the other error in that flow (in the end of the file)